common/sgx/qeidentity.c

```diff
@@ -66,6 +66,7 @@ oe_result_t oe_validate_qe_identity(
     OE_TRACE_INFO(
         "*qe_identity.qe_id_info:[%s]\n",
         sgx_endorsements->items[OE_SGX_ENDORSEMENT_FIELD_QE_ID_INFO].data);
+    platform_tcb_level.isvsvn[0] = qe_report_body->isvsvn;
```

Comment:
Change looks good.  Please add some space and add a comment, something like "Configure the platform isvsvn from the QE report"

Reply:
Fixed

---
In reply to: [376122904](https://github.com/openenclave/openenclave/pull/2510#discussion_r376122904) [](ancestors = 376122904)